index.html

```diff
@@ -1472,6 +1472,193 @@ <h2>
         feature/default allowlist=] is [=default allowlist/'self'=].
       </p>
     </section>
+    <section data-cite="webdriver WebDriver-BiDi">
```

Comment:
From WebDriver perspective the only entry point required in this specification is this algorithm, which stores the "emulated coordinates" somehow and return it if it's set instead of accessing the real system.
```
To <dfn>set emulated coordinates</dfn> given |top-level traversable| and
optional |emulated coordinates| {{GeolocationPosition}}:
```

WebDriver specification will take care of commandsetting it for all the required top-level traversables.